pallets/nft-sales/src/tests.rs

```diff
@@ -391,8 +391,7 @@ fn buy_nft_respects_max_offer_currency() {
 fn prepared_nft(owner: &Origin) -> (u64, ItemId) {
 	let (collection_id, item_id) = (0, ItemId(1));
 
-	// Mint the nft in the uniques pallet
-	assert_ok!(Uniques::create(owner.clone(), collection_id, SELLER));
+	assert_ok!(Uniques::create(owner.clone(), SELLER));
```

Comment:
What is the reason for this test change to not pass a collection_id anymore?

Reply:
Parity changed the function signature:
https://github.com/paritytech/substrate/blob/polkadot-v0.9.28/frame/uniques/src/lib.rs#L466